### ISSUE TITLE
Inventaires - création

### ISSUE THREAD

**Issue by zernerj** [RefLex-Team/RefLex-issues#4]
**Seb, Guillaume, si vous pouvez juste vérifier que ce que je dis ici est pertinent :** 

Le menu inventaire du module management marche bien, pour les inventaires existants.
Pour les inventaires à créer, je n'ai rien trouvé dans l'aide.
Voilà ce que je propose de mettre (en formulant de façon plus impersonnelle) : 

Pour créer un inventaire :
- aller dans l'onglet statistique
- mettre en "active éléments" l'élément dont tu souhaites extraire un inventaire
- cliquer sur le type d'inventaire que tu veux faire en premier (voyelles, consonnes,tons) en cliquant sur les lettres bleues adhoc
- peaufiner ton inventaire selon les options qui s'affichent
- en haut du tableau d'inventaire qui s'affiche, tu vois un petit plus blanc sur vert, et quand tu cliques, ça crée ton inventaire qui s'affiche dans les active elements
- tu peux ici le peaufiner et lui donner un nom, puis valider le tout en cliquant sur "save"

=>  à partir de là, tu le retrouveras, gérable dans l'onglet management/inventories

**Comment by sflavier** [comment]
Oui, ça me semble tout bon. Avec un ou deux visuel, ce sera parfait.
